src/app.php

```diff
@@ -6,11 +6,16 @@
  */
 
 use DrevOps\GitArtifact\Commands\ArtifactCommand;
+use DrevOps\GitArtifact\GitArtifactGit;
 use Symfony\Component\Console\Application;
+use Symfony\Component\Filesystem\Filesystem;
 
 $application = new Application();
 
-$command = new ArtifactCommand();
+$git = new GitArtifactGit();
```

Comment:
please move this into the ArtifactCommand class

please do not modify this file - it is a standard entrypoint

Reply:
Hi @AlexSkrypnyk 
I have reverted to standard entry point.